src/index.js

```diff
@@ -172,16 +174,14 @@ export default {
         width: "100%",
         overflow: "hidden",
       },
-      attrs: {
-        src: this.frameUrl(),
-        allow: "camera",
-        scrolling: "no",
-        referrerpolicy: "no-referrer-when-downgrade",
-      },
+      src: this.frameUrl(),
```

Comment:
I'm not sure if this was due to Vue2->Vue3, but with `attrs` as an object, it was appearing as `attrs='[Object object]'` on the iframe tag, rather than as the individual attributes.

Reply:
Maybe related, but it looks like people were running into a [similar issue](https://github.com/vuejs/router/issues/494#issuecomment-1107639056) with the vue router when upgrading 2 -> 3. Either way your fix seems like the way to go.